### DISCUSSION TITLE
Rst preview

### DISCUSSION THREAD

**Discussion by Lev135** [community/community#39810] [Mobile]
Maybe I'm doing something wrong, bit it seems, that there is no option to open an .rst document in rendered view like as Markdown. It would be very helpful for me, since there are repositories, where text document are in .rst, but not .md

**Comment by ossanna16** [comment]
Hi there @Lev135 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
